environment/integration/repository_test.go

```diff
@@ -0,0 +1,128 @@
+package integration
```

Comment:
Shouldn't these go into `repository/repository_test.go`? We're testing `Open` over there and the rest here here